packages/scss/src/components/progress/states.scss

```diff
@@ -11,15 +11,11 @@
 @mixin success {
 	@include state;
 
-	.progress-bar {
-		background-color: var(--palettes-success-700);
-	}
+	--palettes-700: var(--palettes-success-700);
```

Comment:
There is an inconsistency in LF with palettes states management. Eg some components use `palette-success` while other, like this one use `is-success`.

Without renaming anything, i think that we should echo this into our palettes management (file base.scss) :

```
.palette-success,
.is-success {
  --palette-50: xxx;
  ...
}
```
instead of just
```
.palette-success {
  --palette-50: xxx;
  ...
}
```